x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/monitor_list_drawer/monitor_list_drawer.tsx

```diff
@@ -55,7 +54,7 @@ export function MonitorListDrawerComponent({
     loadMonitorDetails(summary.monitor_id);
   }, []);
 
-  const monitorUrl: string | undefined = get(summary.state.url, 'full', undefined);
+  const monitorUrl: string | undefined = summary.state.url?.full ?? undefined;
```

Comment:
here you also don't need to assign undefined.

Reply:
Same as  https://github.com/elastic/kibana/pull/50966#discussion_r358862326, the `full` field here can be `null`.